src/MICore/LaunchOptions.cs

```diff
@@ -551,6 +551,13 @@ private static string ResolveFromPath(string command)
                 {
                     return candidate;
                 }
+
+                string exeCandidate = candidate + ".exe";
```

Comment:
I don't think this is what you want. I think you want 
```
if (String.IsNullOrEmpty(Path.GetExtension(command)) && PlatformUtilities.IsWindows()) 
{ 
    command.Append(".exe"); 
}
```

above if its windows and doesn't contain a file extension. This code checks what they put in AND whatever they enter + .exe